### ISSUE TITLE
DeprecationWarning: invalid escape sequence '\*'

### ISSUE THREAD

**Issue by drernie** [wbolster/jsonlines#82]
Has this been resolved already?
```
../../../Library/Caches/pypoetry/virtualenvs/udp-spec-hcYzcLID-py3.10/lib/python3.10/site-packages/jsonlines/jsonlines.py:301
  /Users/quilt/Library/Caches/pypoetry/virtualenvs/udp-spec-hcYzcLID-py3.10/lib/python3.10/site-packages/jsonlines/jsonlines.py:301: DeprecationWarning: invalid escape sequence '\*'
    """

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

**Comment by drernie** [comment]
FYI, in case this helps
https://adamj.eu/tech/2022/11/04/why-does-python-deprecationwarning-invalid-escape-sequence/
> [pyupgrade](https://github.com/asottile/pyupgrade/) is a tool for upgrading Python code to the latest syntax. It has a [fixer](https://github.com/asottile/pyupgrade/#invalid-escape-sequences) to rewrite invalid escape sequences automatically.

**Comment by wbolster** [comment]
i think this was fixed in 2019 already: 2ccbfe85c334ad07b15c683f838658c0d89c8b48

please upgrade
